draft-ietf-mls-protocol.md

```diff
@@ -1948,23 +1944,25 @@ provides forward secrecy and post-compromise security with regard to the sender
 of the Commit.  An Update proposal can be regarded as a "lazy" version of this
 operation, where only the leaf changes and intermediate nodes are blanked out.
 
-The `path` field of a Commit message MUST be populated if the Commit covers at
-least one Update or Remove proposal, i.e., if the length of the `updates` or
-`removes` vectors is greater than zero.  The `path` field MUST also be populated
-if the Commit covers no proposals at all (i.e., if all three proposal vectors
-are empty).  The `path` field MAY be omitted if the Commit covers only Add
-proposals.  In pseudocode, the logic for whether the `path` field is required is
-as follows:
+Group members MUST verify that the `proposals` field is sorted such that Update
+proposals come first, then Remove proposals, then Add proposals. The `path`
```

Comment:
It might be good to (1) break these out into bullets, and (2) state explicitly where other proposal types fit (presumably after Adds, in any order defined by the proposals?).

Reply:
As I mentioned on the call today, we need a way for clients to state their support for proposals, e.g., a `supported_proposals` field in the `capabilities` extension.

Reply:
I didn't state where other proposals go, because I thought extensions would want to decide for themselves.

I think support for given proposals should be assumed if the client supports a given group extension